source/slides/10_cfg_mgt.rst

```diff
@@ -2,3 +2,74 @@
 
 Configuration Management History & Basics
 =========================================
+
+
+DevOps workflow in an agency environment
+========================================
+
+Squishymedia
+
+.. image:: /_static/squishy.png
+
+Greg Lund-Chaix
+Director of Technology
+
+
+Tools @ Squishy
+===============
+
+* GitlLab & GitLab CI - http://gitlab.com
+* Puppet - http://puppetlabs.com
+* Vagrant - http://vagrantup.com
+
+Workflow @ Squishy
+==================
```

Comment:
Mind switching this to ``---`` to give the slide more room? I think its fine on your other slides.